CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.9)
+project(DataBase)
+
+include(configure/CMakeLists.txt)
+
+set(MAJOR_VERSION "5")
+set(MINOR_VERSION "3")
```

Comment:
The current version should actually be 5.6.0 as on Sourceforge.
This change was forgotten in the Makefile (which should be updated too) after the preparation of Tango 9.2.5 release which was done from Sourceforge.